src/credentials/credential-wallet.ts

```diff
@@ -367,6 +367,20 @@ export class CredentialWallet implements ICredentialWallet {
               }`
             : `${credentialStatus.id.replace(/\/$/, '')}`
         };
+      case CredentialStatusType.Iden3OnchainSparseMerkleTreeProof2023: {
+        const issuerId = DID.idFromDID(issuer);
+        const chainId = getChainId(DID.blockchainFromId(issuerId), DID.networkIdFromId(issuerId));
+        return {
+          ...credentialStatus,
+          // `[did]:[methodid]:[chain]:[network]:[id]/credentialStatus?(revocationNonce=value)&[contractAddress=[chainID]:[contractAddress]]&(state=issuerState)`
+
+          id: `${issuer.string()}/credentialStatus?revocationNonce=${
+            request.revocationOpts.nonce || ''
+          }&contractAddress=${chainId}:${request.revocationOpts.id}&state=${
```

Comment:
I would not include &state or &revocationNonce at all if corresponding values are empty